README.md

```diff
@@ -58,6 +58,40 @@ long_line_under_100_chars
 # Extra Specs Coding Style
 
 - Add a blank line between `let` and `before`.
+- Group specs around the described object's public methods. Use `describe "#method_name` for instance methods and `describe ".method_name"` for class methods.
+- Context setup does not belong to the `it` block, that's the place for the assert.
+- Context description should always start like `context "when ..."` or `context "and ..."`
+- When you see `it "returns foo when baz is true"` it means you're missing a `context "when baz is true"`
+- Prefer `build_stubbed` over `build` over `create` whenever possible. Helps keep spec suite fast, encourages DI, can prevent N+1 queries just by making you aware of DB queries.
+- When writing factories, don't specify more than what's needed to pass validation, put the more detailed stuff under well named traits.
+- Don't use random values in the factories, that can make specs flaky.
```

Comment:
That means don't use `Faker`... 
I disagree with this rule as fixed value may create false negative.

Reply:
Yeah, we shouldn't use faker in the factory. It is very annoying when a spec fails on CI but you don't know what it's input value was and you can't reproduce it locally.
Faker is good for demo data, I don't see how it's necessary for tests.

> I disagree with this rule as fixed value may create false negative.

What does this mean? You lost me at false negative. 😅

Reply:
I mean't I disagree with the `Don't use random value` rule.
Because flaky test is not bad itself (it's just very annoying), it means that either the code doesn't handle an edge case, either the spec setup is wrong and then should be fixed.

I know this opinion is not shared with everyone so I would not add the `Don't use random value` rule until everyone agree or disagree.